diskimage_builder/elements/ubuntugpu/install.d/51-gpu

```diff
@@ -2,8 +2,8 @@
 
 # Installation of general packages...
 apt-get -y install linux-headers-generic
-apt-get -y install python-pip git software-properties-common python-virtualenv
-python -m pip install --upgrade pip
+apt-get -y install python3-pip git software-properties-common python3-virtualenv
+python3 -m pip install --upgrade pip
```

Comment:
`pip3 install --upgrade pip` ?

Reply:
Pip3 doesn't exist. 
I checked it
`which pip` references /usr/bin/pip which is a wrapper for a call of the pip module using `#!/usr/bin/python3`
which python3 references to `#!/usr/bin/python3`

Reply:
I'll check the installation again.

Reply:
On Bionic, I have this:
```
root@vt-bionic:/home/ubuntu# pip --version
pip 9.0.1 from /usr/lib/python2.7/dist-packages (python 2.7)
root@vt-bionic:/home/ubuntu# pip3 --version
pip 9.0.1 from /usr/lib/python3/dist-packages (python 3.6)
```

Reply:
On focal, I have the selection:
```
root@focal:/home/ubuntu# python
python2    python2.7  python3    python3.8
```
So there is no /usr/bin/python anymore (which was mapped to 2.7 as far as I know)

Reply:
Installing `apt-get -y install python3-pip`
leads to only having a pip3 (no pip)

Then calling:
```
root@focal:/home/ubuntu# python3 -m pip install --upgrade pip
Collecting pip
  Downloading pip-20.1-py2.py3-none-any.whl (1.5 MB)
...
Installing collected packages: pip
  Attempting uninstall: pip
    Found existing installation: pip 20.0.2
    Not uninstalling pip at /usr/lib/python3/dist-packages, outside environment /usr
    Can't uninstall 'pip'. No files were found to uninstall.
Successfully installed pip-20.1
```

Reply:
The pip leads to:
```
root@focal:/home/ubuntu# which pip
/usr/local/bin/pip
root@focal:/home/ubuntu# cat /usr/local/bin/pip
#!/usr/bin/python3
# -*- coding: utf-8 -*-
import re
import sys
from pip._internal.cli.main import main
if __name__ == '__main__':
    sys.argv[0] = re.sub(r'(-script\.pyw|\.exe)?$', '', sys.argv[0])
    sys.exit(main())
```

Reply:
Ok should work then